modules/ROOT/pages/manage-backup-restore.adoc

```diff
@@ -1,63 +1,93 @@
+
+[WARNING]
+====
+Configuration changes to deployed applications and management services made after a back up cannot not be restored.  
+====
 
 [NOTE]
-The state of Runtime Fabric is distributed across its controller nodes. Use the following procedures to back up and restore
-the required state necessary for all deployed applications, management services, and their configurations. 
-Individual node backup and restore operations are not supported.
+====
+For Runtime Fabric running on VMs / Bare Metal, the backup process also includes backing up of underneath Kubernetes appliance related custom resources [logforwarder, smtp, alert and alerttarget].  
+====
 
 
-== Prerequisites for Runtime Fabric on VMs / Bare Metal
+== Prerequisites for Runtime Fabric
 
-Before performing a backup or restore operation, verify that Runtime Fabric on VMs / Bare Metal was installed or upgraded 
-using  https://docs.mulesoft.com/release-notes/runtime-fabric/runtime-fabric-installer-release-notes[installer version 1.1.1581474166, role=external, window=_blank] or later, and that the rtfctl CLI tool is upgraded to version 0.2.95 or later. The versions can be verified by running `rtfctl version` on a Runtime Fabric node.
+Before performing a backup or restore operation:
+
+* Verify that Runtime Fabric is installed and running successfully.
+* Verify that `rtfctl` CLI tool is upgraded to version 0.3.95 or later. Verify the version by running `rtfctl version` on a Runtime Fabric node.
+* If you are using Runtime Fabric on VMs / Bare Metal, verify that Runtime Fabric is upgraded to version 1.1.1581474166 or later. See xref:release-notes::runtime-fabric/runtime-fabric-installer-release-notes.adoc[Anypoint Runtime Fabric Installer Release Notes].
 
 == Create a Backup
 
-To create a backup:
+To create a backup run the following command:
 
-. Run the following command on any controller node in the cluster as a privileged user (with root access):
-+
 ----
 ./rtfctl backup <path_to_backup_file>
 ----
+
 This command creates an archive of the current system state in `<path_to_backup_file>`, which can be any path in the file system that you have write access to, for instance `/opt/anypoint/runtimefabric/backup.tar.gz`. 
 
-. Move the archive file to secure storage (a different machine that is not part of the RTF cluster);
-for example:
-+
+[NOTE]
+====
+For Runtime Fabric running on VMs / Bare Metal, The `rtfctl` binary is present in directory `/opt/anypoint/runtimefabric/`. Confirm that you run the backup command on any controller node in the cluster as a privileged user. To use the archive for restoring later, Copy the archive file to secure storage that is on different machine and is not part of the RTF cluster. The following command, for example, uses `scp` to copy the archive to another drive: 
```

Comment:
lower case "the" after "Bare Metal,"

Reply:
also, lower case "Copy" in "copy the archive file"
"on a different machine" (missing article 'a')

Reply:
In this context "Runtime Fabric running on VMs / Bare Metal" is the official product name. It should not be lower case.

Reply:
RTF should be Runtime Fabric.

Reply:
Runtime Fabric running on VMs / Bare Metal must be changed to `Runtime Fabric on VMs / Bare Metal` which is the official product name.